src/pybind/mgr/dashboard/controllers/prometheus.py

```diff
@@ -30,20 +30,22 @@ class PrometheusRESTController(RESTController):
     def prometheus_proxy(self, method, path, params=None, payload=None):
         # type (str, str, dict, dict)
         return self._proxy(self._get_api_url(Settings.PROMETHEUS_API_HOST),
-                           method, path, 'Prometheus', params, payload)
+                           method, path, 'Prometheus', params, payload,
+                           Settings.PROMETHEUS_API_SSL_VERIFY)
```

Comment:
Minor thing: just for the sake of clarity (a method with 7 positional arguments starts being hard to follow), I'd add this a as keyword argument:
```suggestion
                           verify=Settings.PROMETHEUS_API_SSL_VERIFY)
```